src/Activators/FileActivator.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Nwidart\Modules\Activators;
+
+use Illuminate\Cache\CacheManager;
+use Illuminate\Config\Repository as Config;
+use Illuminate\Container\Container;
+use Illuminate\Filesystem\Filesystem;
+use Nwidart\Modules\Contracts\ActivatorInterface;
+use Nwidart\Modules\Module;
+
+class FileActivator implements ActivatorInterface
+{
+	/**
+	 * Laravel cache instance
+	 * 
+	 * @var CacheManager
+	 */
+	protected $cache;
+
+	/**
+	 * Laravel Filesystem instance
+	 * 
+	 * @var Filesystem
+	 */
+	protected $files;
+
+	/**
+	 * Laravel config instance
+	 * @var Config
+	 */
+	protected $config;
+
+	/**
+	 * @var string
+	 */
+	protected $cacheKey;
+
+	/**
+	 * @var string
+	 */
+	protected $cacheLifetime;
+
+	/**
+	 * Array of modules activation statuses
+	 * 
+	 * @var array
+	 */
+	protected $installed;
```

Comment:
Could this be renamed to `$installedModules`. I was hesitating if this was a boolean or not. 

Also what's an "installed module"? Is it an active module? If so, it might be more clear to use `activeModules` instead.

Reply:
The way I see it, an installed module is a module that exists on the disk, it then has an activation status (1 or 0), both are stored by this activator.

Reply:
Hm, thanks that explains the keyword a bit. So this contains an array of all modules? If so I'd still rename it `installedModules` to be explicit.